examples/complete/main.tf

```diff
@@ -132,6 +132,8 @@ module "k8s-devsecops-sandbox" {
   hosted_zone_id           = var.hosted_zone_id
   elb_dns_name             = module.rancher-k8s-cluster.elb_dns_name
   elb_zone_id              = module.rancher-k8s-cluster.elb_zone_id
+  jenkins_admin_email      = var.jenkins_admin_email
+  jenkins_host_name        = "jenkins.${random_pet.default.id}.${var.hosted_zone_domain_name}"
```

Comment:
Does this need to reference var.jenkins_host_name instead so it is set from a tfvars?

Reply:
Normally yes, but since this is an example, and it is creating a unique host name each time via the `random_pet` resource, it needs to be set this way rather than just passing it through as a variable.